public/data/mainfunktion.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "Sean-michael Chambers",
+  "type": "personal",
+  "bio": "Software engineering student currently learning python and solving kata's on codewars.",
+  "avatar": "https://github.com/mainfunktion.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/mainfunktion",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/0x6f0x67",
+      "icon": "twitter"
+    },
+    {
+      "name": "Add on me linkedin",
```

I recommend capitalizing `linkedin`. What do you think?
```suggestion
      "name": "Add on me LinkedIn",
```

---

Yeah true, I missed that.